wrappers/python/simtk/openmm/app/topology.py

```diff
@@ -404,3 +436,32 @@ def __init__(self, name, element, index, residue, id):
 
     def __repr__(self):
         return "<Atom %d (%s) of chain %d residue %d (%s)>" % (self.index, self.name, self.residue.chain.index, self.residue.index, self.residue.name)
+
+class Bond(tuple):
```

Comment:
Wow, will this really work? So you can still say
```python
for (atom1, atom2) in topology.bonds():
   # do stuff
```
but you could also say
```python
for bond in topology.bonds():
   [atom1, atom2] = bond
   bond_order = bond.order
   bond_type = bond.type
```
Is that right?